rxnetty-http/src/main/java/io/reactivex/netty/protocol/http/server/HttpServerResponseImpl.java

```diff
@@ -52,7 +52,7 @@ private HttpServerResponseImpl(final State<C> state) {
         super(new OnSubscribe<Void>() {
             @Override
             public void call(Subscriber<? super Void> subscriber) {
-                state.sendHeaders().write(Observable.<C>empty()).unsafeSubscribe(subscriber);
+                state.sendHeaders().unsafeSubscribe(subscriber);
             }
```

Comment:
Turns out this is `write` unnecessary - this was causing differing behaviour between 

```
public Observable<Void> handle(HttpServerRequest<ByteBuf> request, HttpServerResponse<ByteBuf> response) {
    return response;
}
```
and

```
public Observable<Void> handle(HttpServerRequest<ByteBuf> request, HttpServerResponse<ByteBuf> response) {
    return response
        .sendHeaders();
}
```

the former previously returned with an empty chunk, the latter `content-length: 0`